InferenceSystem/README.md

```diff
@@ -1,26 +1,32 @@
-# Run inference locally
+# Working with the InferenceSystem
 
-Using Python 3 specifically tested with Python version 3.7.4
+The InferenceSystem is an umbrella term for all the code used to stream audio from Orcasound's S3 buckets, perform inference on audio segments using the deep learning model and upload positive detections to Azure. The entrypoint for the InferenceSystem is [src/LiveInferenceOrchestrator.py](src/LiveInferenceOrchestrator.py).
 
+This document describes the following steps
+1. How to run the InferenceSystem locally.
+2. Deploying an updated docker build to Azure Container Instances.
+
+Note: We use Python 3, specifically tested with Python 3.7.4
+
+# How to run the InferenceSystem locally
 ## Create a virtual environment
 
 1. In your working directory, run `python -m venv inference-venv`. This creates a directory `inference-venv` with relevant files/scripts. 
 2. On Mac, activate this environment with `source inference-venv/bin/activate` and when you're done, `deactivate`
+
     On Windows, activate with `.\inference-venv\Scripts\activate.bat` and `.\inference-venv\Scripts\deactivate.bat` when done
 3. In an active environment, cd to `/InferenceSystem` and run `python -m pip install --upgrade pip && pip install -r requirements.txt` 
 
 ## Model download
 
-1.  Download model from https://microsoft.sharepoint.com/:u:/t/OrcaCallAutomatedRecognitionSystemHackathon2019Project/EV9IBJrfmOxKhWtZOaW3pB8Br5u3yF0K3L18eZDruw89jw?e=S8UyEC
-2.  Unzip model.zip and extract to `InferenceSystem/model`
-3.  Check the contents of InferenceSystem/model
-There should be 3 files
-    Audioset_fc_all_*
-    mean64.txt
-    invstd64.txt
+1.  Download the current production model from [this link.](https://trainedproductionmodels.blob.core.windows.net/dnnmodel/11-15-20.FastAI.R1-12.zip)
+2.  Unzip *.zip and extract to `InferenceSystem/model`.
+3.  Check the contents of `InferenceSystem/model`.
+There should be 1 file
+    * model.pkl
 
 ## Get connection string for interface with Azure Storage
-
+To be able to upload detections to Azure you will need a few strings.
 1.  Go to the [Azure portal](https://portal.azure.com/)
 Go to the `"LiveSRKWNotificationSystem"` resource group and within that go to the "livemlaudiospecstorage" storage account. Refer to [this page](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-python#copy-your-credentials-from-the-azure-portal) to see how to get the connection string.
```

Comment:
```suggestion
Go to the [Azure portal](https://portal.azure.com/). Go to the `"LiveSRKWNotificationSystem"` resource group and within that go to the "livemlaudiospecstorage" storage account. Refer to [this page](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-python#copy-your-credentials-from-the-azure-portal) to see how to get the connection string.
```
There's a similar snippet with the same issue below in the "Get primary key for interface with CosmosDB" section.